st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -48,6 +52,17 @@ class ActionAliasController(resource.ContentPackResourceController):
         'sort': ['pack', 'name']
     }
 
+    _custom_actions = {
```

Comment:
There we go - 4afaa54f7e980a2fd5d66c9f82dda127b4387366

It works now, woo!

```bash
(virtualenv)vagrant@vagrant-ubuntu-trusty-64:/data/stanley$ python ./st2client/st2client/shell.py action-alias match "run uptime on localhost"
2016-09-15 12:37:32,670  WARNING - Auth API server is not available, skipping authentication.
+------------------+---------------------------------------------+
| name             | description                                 |
+------------------+---------------------------------------------+
| remote_shell_cmd | Execute a command on a remote host via SSH. |
+------------------+---------------------------------------------+
(virtualenv)vagrant@vagrant-ubuntu-trusty-64:/data/stanley$ python ./st2client/st2client/shell.py action-alias execute "run uptime on localhost"
2016-09-15 12:37:34,473  WARNING - Auth API server is not available, skipping authentication.
Started execution, id '57da960e0640fd107ab26409'
Hold tight while I run command: *uptime* on hosts *localhost*
```

Minor thing - what do other things, I think `execute` could be blocking by default (similar as `st2 run` command) and optionally user can specify `--async` flag to make it non-blocking. WDYT?